modules/test/playwright/tests/object-web/main/objectFields.spec.ts

```diff
@@ -757,6 +757,64 @@ test.describe('Manage object fields through Model Builder', () => {
 });
 
 test.describe('Manage objectFields through Objects Admin UI', () => {
+	test("can check that it's possible to update created custom fields in a System Object", async ({
+		apiHelpers,
+		objectFieldsPage,
+		page,
+	}) => {
+		const objectDefinitionField =
+			await apiHelpers.buildRestClient(ObjectFieldAPI);
+
+		const fieldName = 'Custom Field';
+
+		const {items} = await apiHelpers.objectAdmin.getAllObjectDefinitions();
+
+		const systemObjectDefinition = items.find((item: ObjectDefinition) => {
+			return item.system === true;
+		});
+
+		await objectDefinitionField.postObjectDefinitionObjectField(
+			systemObjectDefinition.id,
+			{
+				DBType: 'String',
+				businessType: 'Text',
+				indexed: true,
+				label: {en_US: fieldName},
+				localized: false,
```

Comment:
Could you check if the located property is necessary? I don't think so. You can take the opportunity to check if there are other unnecessary properties in this object.